apps/system/utils/utils_proc.h

```diff
@@ -22,7 +22,7 @@
 #include <tinyara/config.h>
 #include <dirent.h>
 
-#if defined(CONFIG_ENABLE_IRQINFO) || defined(CONFIG_ENABLE_IRQINFO) || defined(CONFIG_ENABLE_PS) || defined(CONFIG_ENABLE_STACKMONITOR) || defined(CONFIG_ENABLE_HEAPINFO)
+#if defined(CONFIG_ENABLE_IRQINFO) || defined(CONFIG_ENABLE_IRQINFO) || defined(CONFIG_ENABLE_PS) || defined(CONFIG_ENABLE_STACKMONITOR) || defined(CONFIG_ENABLE_HEAPINFO) || defined(CONFIG_ENABLE_CPULOAD)
```

Comment:
I don't know why this is necessary. Could you check the operation without this conditional?